steveify1/nrc-hr-backend
# #247 chore(migrations):create and seed tables

#### What does this PR do?
create tables and models, and seed tables
#### Description of Task to be completed?
- create and seed `CareerReasonCodes` table
- create and seed `IncidenceReasonCodes` table
- create `Careers` table and `Career` model
- create `JobIncidence` table and model
#### How should this be manually tested?
* clone the repository
* cd into the project directory
* run `npm install`
* run `npm test`
* check the `models`, `migrations` and `seeders` to see new files
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
#247 #245 #246 #244 #239
#### Screenshots (if appropriate)
#### Questions:
